### PR TITLE
Keadm: delete extra residual files

### PR DESCRIPTION
**Pull request by TianTianBigWang**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by fisherxu** [comment]
/approve
@TianTianBigWang Thanks for the fixing!

File: keadm/cmd/keadm/app/cmd/reset.go [hunk 1]
```diff
@@ -160,7 +160,12 @@ func RemoveContainers(isEdgeNode bool, execer utilsexec.Interface) error {
 }
 
 func cleanDirectories(isEdgeNode bool) error {
-	var dirToClean = []string{"/var/lib/edged", "/etc/kubeedge"}
+	var dirToClean = []string{
+		"/var/lib/edged",
```
Comment (daixiang0) [inline keadm/cmd/keadm/app/cmd/reset.go:164]: try use existed const in keadm

File: keadm/cmd/keadm/app/cmd/reset.go [hunk 2]
```diff
@@ -160,7 +160,12 @@ func RemoveContainers(isEdgeNode bool, execer utilsexec.Interface) error {
 }
 
 func cleanDirectories(isEdgeNode bool) error {
-	var dirToClean = []string{"/var/lib/edged", "/etc/kubeedge"}
+	var dirToClean = []string{
+		util.KubeEdgePath,
+		util.KubeEdgeLogPath,
+		"/var/lib/edged",
```
Comment (daixiang0) [inline keadm/cmd/keadm/app/cmd/reset.go:166]: If no const, let's add it, same as below

**Pull request merged**